src/utils/Image.js

```diff
@@ -15,6 +15,19 @@ export const scaleHeight = ({width, height}, scaleWidth) => (
   }
 );
 
+/** Function to retrurn the heights of the post images 
+  * @return {Number} - The height of the image post
+  */
+export const getPostImageHeight = () => {
+  const width = getWindowDimensions().width;
+  switch (width) {
+    case 320: return 200;
+    case 375: return 225;
+    case 414: return 250;
+    default: return 250;
+  }
+}
+
```

Comment:
Image height needs to be calculated based on device width so as to avoid this:
![simulator screen shot 02 sep 2016 3 22 10 pm](https://cloud.githubusercontent.com/assets/9957046/18215829/466a5386-7121-11e6-9c7c-2e03f7ebb7ba.png)

(Could not find StyleSheet solution using https://github.com/facebook/react-native/issues/950)